modules/apps/shared-dependencies/shared-dependencies-groovy-all/bnd.bnd

```diff
@@ -0,0 +1,71 @@
+Bundle-Name: Liferay Shared Dependencies Groovy All
+Bundle-SymbolicName: com.liferay.shared.dependencies.groovy.all
+Bundle-Version: 1.0.0
+Import-Package:\
+	!com.thoughtworks.qdox.*,\
+	!com.thoughtworks.xstream.*,\
+	\
+	!groovyjarjarasm.asm.tree.analysis.*,\
+	\
+	!jline.*,\
+	\
+	!junit.*,\
+	\
+	!org.apache.bsf.*,\
+	!org.apache.commons.cli.*,\
+	!org.apache.ivy.*,\
+	!org.apache.tools.*,\
+	\
+	!org.fusesource.jansi.*,\
+	\
+	!org.junit.*,\
+	\
+	*
+-exportcontents:\
+	groovy.beans,\
+	\
+	groovy.cli.*,\
+	\
+	groovy.grape,\
+	\
+	groovy.inspect.*,\
+	\
```

Comment:
@marianoalvarosaiz, It lacks of `groovy.inspect.swingui` in the export-package. Please check again.

Reply:
groovy.inspect.* references groovy.inspect.swingui, I've tested in a Groovy script.